infra/common/src/test/java/cn/hippo4j/common/executor/ThreadFactoryBuilderTest

```diff
@@ -0,0 +1,71 @@
+package cn.hippo4j.common.design.builder;
```

Comment:
Add open source protocol

Reply:
what is open source protocol,I can't see it in other pr

Reply:
<img width="1722" alt="image" src="https://github.com/opengoofy/hippo4j/assets/77398366/a302b2a0-ed8d-400a-928d-c720680a3222">

Open any class to see the top

Reply:
Thanks